packages/oidc/src/dpop/tokenExchange.ts

```diff
@@ -222,20 +221,22 @@ export async function getTokens(
     )}`;
   }
 
+  const requestBody = {
```

Comment:
This follows the same style as in `refreshGrant.ts` where the body is constructed separately before being passed to `new URLSearchParams`